bug-bounty-list/README.md

```diff
@@ -0,0 +1,57 @@
+# disclose.io/bug-bounty-list
+
+> The community-powered database of ALL known public bug bounty and vulnerability programs, including their safe-harbor status.  
+
+## How to contribute  
+If you've found a new program, new information on an existing program, or think a program should be deleted, you can contribute to bug-bounty-list by issuing a pull request (PR) to the repo. If you're new to Github, [this article](https://www.thinkful.com/learn/github-pull-request-tutorial/#Time-to-Submit-Your-First-PR) is a good primer on how PRs work. 
+
+## Before you go hunting for new programs to add...
+
+Programs on bug-bounty-list need to satisfy the definition of a public bug bounty or vulnerability disclosure program, which means they need two key components:
+  1. A publicly accessible vulnerability disclosure policy sometimes called a program brief, or bounty brief, and 
+  2. A publicly accessible channel for vulnerability submission **which is explicitly mentioned in the vulnerability disclosure policy**.  
+  
+Some examples of suggestions that won't be accepted:
+  1. Private programs  
+  2. Invitation-only programs  
+  3. Programs that you heard about from a friend but aren't listed publicly  
+  4. Programs without a public policy or a nominated channel for communication  
+
+## Definition of fields 
+
+**program_name (required):** What is the name of the company or product team responsible for the program?  
+**policy_url (required):** Where can the policy which defines the program be found? (Usually a webpage e.g. /security, a URL on a vulnerability management platform, or in a robots.txt or security.txt file).  
+**submission_url (required):** Where does policy say vulnerability submissions should be sent? (Usually a web form, a URL on a vulnerability management platform, or an email address. Emails can be added as URLs using 'mailto:').  
+**launch_date (optional):** When does the program say it was launched?  
+**cash_reward (optional):** Does the program commit to rewarding vulnerabilities?  
+**swag (optional):** Does the program commit to sending swag out for vulnerabilities?  
+**hall_of_fame (optional):** Does the program have a listing or Hall of Fame page acknowledging vulnerability reporters . 
+**safe_harbor (required):** Does the program support full or partial safe-harbor?  
```

Comment:
You may want to refer to the root directory README, which briefly outlines what "partial" safe harbor means.

Reply:
Ack. We should extend the definitions in the master docs as well.